packages/tss-client/src/client.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console */
 import { generatePrivate } from "@toruslabs/eccrypto";
-import type TssLib from "@toruslabs/tss-dkls-lib";
+import type { WasmLib } from "@toruslabs/tss-dkls-lib";
```

Comment:
Hmm, do we need to change this?

Reply:
Not necessarily, but I think having a type is actually what we want here. Then we don't need to write `typeof TssLib` every time.